packages/wonder-blocks-tokens/src/theme/semantic/internal/primitive-color-thunderblocks.ts

```diff
@@ -0,0 +1,109 @@
+import {fade} from "../../../util/utils";
+
+// Base colors
+const black = "#191918";
```

Comment:
My one thought on working through some font theming is: should the default theme be put into these theme folders to keep similar objects near each other in the codebase? I think it could help reduce some cognitive complexity for theming.